test/Benchmarks/src/Vector/Distinct.enso

```diff
@@ -1,31 +1,27 @@
 from Standard.Base import all
-import Standard.Base
 
 from Standard.Test import Bench
 
 import project.Vector.Utils
 
-polyglot java import org.enso.base.Time_Utils
 
+options = Bench.options . size 100 . iter 20
 
-## Bench Utilities ============================================================
+random_vec = Utils.make_random_vec 100000
+uniform_vec = Vector.fill 100000 1
+random_text_vec = random_vec.map .to_text
+uniform_text_vec = random_vec.map .to_text
```

Comment:
This is not good.

Enso has a subtle difference between top-level and scoped definitions, that plays a very crucial role here.

A scoped definition is computed (unless it's a block...) and stored. A top-level definition is re-computed on every access.

Essentially you have turned values _computed once at initialization of the suite_ into 0-argument methods that are _recomputed on each access_.

This changes the semantics of these benchmarks.

We used to be just computing the time needed to compute the `distinct` operation. Now we are computing the total time it takes to both generate the vector _and_ compute distinct.

I don't think that's right.

I know that we don't have the 'setup' pattern yet. For now I'd just run this inside of the `group`. It's not perfect and will slow down gathering benchmarks, so maybe for future we need a better solution. 

On the other hand, I'm not 100% convinced that changing semantics is bad here. I don't think it is good but feel free to convince me otherwise.

Reply:
On second thought, let me take care of this in follow-up PRs, I have now added it to a task list in #7489.